src/lib/shady-render.ts

```diff
@@ -277,6 +277,7 @@ export const render =
       litRender(
           result,
           renderContainer,
+          // eslint-disable-next-line
```

Comment:
Huh, what does eslint dislike about this line?

Reply:
Huummm... I guess I lost it after a bad update. Updating!